.gitignore

```diff
@@ -53,3 +53,5 @@ local.properties
 # lib files
 ext/SDL2/*.zip
 ext/SDL2/*.tar.gz
+
+res/augustus.icns
```

Comment:
Will this cause problems?

It is build as part of the make command (at least on Linux), so I figure it should be excluded from source.

Reply:
I agree with this.